### PR TITLE
Added timeouts to ensure tests pass

### PR DESCRIPTION
**Pull request by fearphage**
Tests were timing out. I updated the mocha timeout values to allow them to complete.

This was a result of the discussion in #1043.

### CODE REVIEW

File: test/util/fake-timers-test.js [hunk 1]
```diff
@@ -414,6 +416,8 @@ describe("sinon.clock", function () {
         });
 
         it("passes 2 hours, 34 minutes and 12 seconds", function () {
+            this.timeout(50000);
```
Comment (mantoni) [inline test/util/fake-timers-test.js]: This seems a bit excessive. Why does it take almost a minute?

Comment (fearphage) [inline test/util/fake-timers-test.js]: I agree. The problem from a quick glance seems to be based on the implementation of [`clock.tick` in lolex](https://github.com/sinonjs/lolex/blob/master/src/lolex-src.js#L479-L524). Here is [some (very little) context](https://github.com/sinonjs/sinon/pull/1043#issuecomment-222141239). I haven't put a lot of time into investigating though.

**Comment by mantoni** [comment]
It’s better to have the tests pass after a long time instead of a failure where nothing is really wrong, so I merged. The browser builds are working again - let’s see whether they turn green now ⌛

**Comment by mantoni** [comment]
As green as can be! 🎉

**Comment by mroderick** [comment]
🎆

**Comment by mroderick** [comment]
Thanks @fearphage!!!

**Pull request merged**